_includes/releases/v21.2/v21.2.14.md

```diff
@@ -0,0 +1,87 @@
+- Fixed a bug that may have caused a panic if a [Kafka](../v21.2/changefeed-sinks.html#kafka) server being written to by a [changefeed](../v21.2/change-data-capture-overview.html) failed at the wrong moment. [#83922][#83922]
+- The [`cockroach debug merge-logs`](../v21.2/cockroach-debug-merge-logs.html) command no longer returns an error when the log decoder attempts to parse older logs. [#83748][#83748]
+- A Flush message sent during portal execution in the `pgwire` extended protocol no longer results in an error. [#83954][#83954]
+- Previously, [virtual computed columns](../v21.2/computed-columns.html) which were marked as `NOT NULL` could be added to new [secondary indexes](../v21.2/indexes.html). Now, attempts to add such columns to a secondary index will result in an error. Note that such invalid columns can still be added to tables. Work to resolve that bug is tracked in [#81675](https://github.com/cockroachdb/cockroach/issues/81675). [#83552][#83552]
+- Fixed a rare issue where the failure to apply a [Pebble](../v21.2/architecture/storage-layer.html#pebble) manifest change (typically due to block device failure or unavailability) could result in an incorrect [LSM](../v21.2/architecture/storage-layer.html#log-structured-merge-trees) state. Such a state would likely result in a panic soon after the failed application. This change alters the behavior of Pebble to panic immediately in the case of a failure to apply a change to the manifest. [#83734][#83734]
+- Moved connection OK log and metric to the same location after authentication completes for consistency. This resolves an inconsistency (see linked issue) in the [DB Console](../v21.2/ui-overview.html) where the log and metric did not match. [#84175][#84175]
+- Previously, CockroachDB would not normalize `timestamp/timestamptz - timestamp/timestamptz` like PostgreSQL does in some cases (depending on the query). This is now fixed. [#84002][#84002]
+- Fixed an internal error `node ... with MaxCost added to the memo` that could occur during planning when calculating the cardinality of an outer join when one of the inputs had 0 rows. [#84381][#84381]
+- Fixed a bug in transaction conflict resolution which could allow backups to wait on long-running transactions. [#83905][#83905]
+- Fixed a "fake" memory accounting leak that in rare cases could result in `memory budget exceeded` errors even if the actual RAM usage is within `--max-sql-memory` limit. [#84325][#84325]
+- Fixed the following built-in functions so that users can only run them if they have [`SELECT`](../v21.2/select-clause.html) privileges on the relevant tables: `crdb_internal.revalidate_unique_constraints_in_all_tables`, `crdb_internal.revalidate_unique_constraints_in_table`, and `crdb_internal.revalidate_unique_constraint`. [#84272][#84272]
```

Comment:
Link to functions-and-operators.html#built-in-functions here on "built-in functions"?